components/automatic-points-table/commons/automatic-points-table.lua

```diff
@@ -112,4 +117,51 @@ function AutomaticPointsTable:parseManualPoints(team, tournamentCount)
 	return manualPoints
 end
 
+function AutomaticPointsTable:queryPlacements(tournaments)
+	local queryParams = {
+		limit = 5000,
+		query = 'tournament, participant, placement, extradata'
+	}
+
+	local tournamentNames = Table.mapValues(tournaments,
+		function(tournament)
+			return tournament.name
+		end
+	)
+
+	local conditions = '[[tournament::' ..
+		table.concat(tournamentNames, ']] OR [[tournament::') ..
+		']]'
+
+	queryParams.conditions = conditions
+	local allQueryResult = mw.ext.LiquipediaDB.lpdb('placement', queryParams)
+
+	local tournamentsWithPlacements = Table.mapValues(tournaments,

```

Comment:
See my comment in #948

Reply:
Noted, please check https://github.com/Liquipedia/Lua-Modules/pull/949#discussion_r783995516